packages/kbn-apm-synthtrace/src/scenarios/helpers/time_based_pattern.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export function timeBasedPattern({
```

Comment:
Should we add a few unit tests on this?

Reply:
I don't particularly care about the correctness of this function, it's just to generate some real-ish looking patterns. If it breaks we'll see soon enough